app/services/atmosphere/flavor_updater_ext.rb

```diff
@@ -0,0 +1,12 @@
+module Atmosphere::FlavorUpdaterExt
+  extend ActiveSupport::Concern
+
+  # By default do nothing. It can be extendend in target project and create
+  # additional objects (`FlavorOsFamily`) with concreate prices. Please don't
+  # save flavor or other created objects, since it will be done at the end of
+  # flavor update operation.
+  #
+  # Flavor `cpu`, `memory` and `hdd` attributes are already set.
+  def calculate_price(flavor)
```

Comment:
Unused method argument - `flavor`. If it's necessary, use `_` or `_flavor` as an argument name to indicate that it won't be used. You can also write as `calculate_price(*)` if you want the method to accept any arguments but don't care about them.